pkg/testing/integration/program.go

```diff
@@ -190,6 +190,9 @@ type ProgramTestOptions struct {
 	// SkipStackRemoval indicates that the stack should not be removed. (And so the test's results could be inspected
 	// in the Pulumi Service after the test has completed.)
 	SkipStackRemoval bool
+	// SkipLifeCycleDestroy indicates we'll skip destroying the resources and stack
+	// Tests that indicate this must manually clean by invoking TestLifecycleDestroy
+	SkipLifeCycleDestroy bool
```

Comment:
It looks like this flag is no longer necessary with the manual life cycle management approach.